.pyup.yml

```diff
@@ -17,7 +17,7 @@ search: True
 # default: empty
 # allowed: list
 requirements:
-  - {{cookiecutter.project_name}}/requirements_dev.txt:
+  - "{{cookiecutter.project_name}}/requirements_dev.txt":
```

Comment:
Not sure this will satisfy pyup...

Reply:
As you mentioned you can't validate that before actually merging, so I don't see much choice here.